src/main/java/com/based/basedsurvey/controller/FillSurveyController.java

```diff
@@ -0,0 +1,159 @@
+package com.based.basedsurvey.controller;
+import com.based.basedsurvey.data.*;
+import com.based.basedsurvey.repo.SurveyRepository;
+import lombok.extern.java.Log;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import java.util.List;
+
+@Log
+@Controller
+public class FillSurveyController {
+
+    private SurveyRepository surveyRepository;
+
+    @Autowired
+    public FillSurveyController(SurveyRepository surveyRepository) {
+        this.surveyRepository = surveyRepository;
+    }
+
+    @GetMapping("/survey/{surveyID}/answer")
```

Comment:
Does the curly braces mean the parameter is optional or is it a variable. I'm guessing it's the latter and in that case looks good